### PR TITLE
docs: Explain about the `systemctl enable` trap

### PR DESCRIPTION
**Pull request by cgwalters**

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -4,6 +4,44 @@ nav_order: 3
...
+systemd presets are designed for "run once" semantics - thereafter, OS upgrades
+won't cause new services to start.  In contrast, "static enablement" by creating
+the symlink (as is done above) bypasses the preset logic.
+
+In general, it's recommended to follow the "static enablement" approach because
+it more closely aligns with "immutable infrastructure" model.
+
+### Using presets
+
+If nevertheless you want to use presets instead of "static enablement", One
```
Comment (vrothberg) [inline docs/usage.md:32]:
```suggestion
If nevertheless you want to use presets instead of "static enablement", one
```

File: docs/usage.md [hunk 2]
```diff
@@ -4,6 +4,44 @@ nav_order: 3
...
+recommended pattern to avoid this problem (nd is also somewhat of a best
```
Comment (vrothberg) [inline docs/usage.md:33]:
```suggestion
recommended pattern to avoid this problem (and is also somewhat of a best
```

File: docs/usage.md [hunk 3]
```diff
@@ -4,6 +4,44 @@ nav_order: 3
...
+practice anyways) is to use a common prefix (e.g. `examplecorp-` for all of your
+custom systemd units), resulting in `examplecorp-checkin.service`,
+`examplecorp-agent.service` etc.
+
+Then you can write a single systemd preset file to e.g.
+`/usr/lib/systemd/system-preset/50-examplecorp.preset` that contains:
```
Comment (vrothberg) [inline docs/usage.md:39]: Is that what you recommend?  If so, I think we should mark/highlight it as such in the section title, and probably move it above the workaround to make sure users see it _before_ the workaround.

Comment (cgwalters) [reply docs/usage.md:39]: The doc just above this is recommending against presets, right?

Comment (vrothberg) [reply docs/usage.md:39]: Fair point.  I must have gotten distracted for a moment. Thanks!

**Review by vrothberg — APPROVED** [review]
LGTM

**Pull request merged**